### PR TITLE
Add --dry-run option to update command

### LINKED ISSUE #1: Add "--dry-run" option to update command
This would be usefull if an application wants to show the user what would be done before actually doing it.

### PR DESCRIPTION
**Pull request by mjauvin**
Fix #1 

Add `getRawOutput()` method to fetch rawOutput when using --dry-run

### CODE REVIEW

**Comment by bennothommo** [comment]
@mjauvin looks good to me. Could you just add a unit test for this, making sure that the output is available and also ensuring that no `composer.lock` file or `vendor` directory is generated?

**Comment by mjauvin** [comment]
@bennothommo when you say "making sure that the output is available and also ensuring that no `composer.lock` file or `vendor` directory is generated" you mean in the unit test, right?

**Comment by bennothommo** [comment]
@mjauvin that's right :)

File: src/Commands/Update.php [hunk 1]
```diff
@@ -145,6 +152,10 @@ public function execute()
 
         $this->rawOutput = $output['output'];
```
Comment (mjauvin) [inline src/Commands/Update.php]: @bennothommo I am skipping the command output parsing in `dry-run` mode. do you think we should still parse the output and see what packages WOULD be installed/removed/updated ? 

(see below)

**Comment by mjauvin** [comment]
> @mjauvin looks good to me. Could you just add a unit test for this, making sure that the output is available and also ensuring that no `composer.lock` file or `vendor` directory is generated?

Done!

File: src/Commands/Update.php (hunk 1 above)
Comment (bennothommo) [reply src/Commands/Update.php]: Yeah I think we'd have to, @mjauvin. That's the entire point of running a dry run 😜

**Comment by mjauvin** [comment]
> > @mjauvin looks good to me. Could you just add a unit test for this, making sure that the output is available and also ensuring that no `composer.lock` file or `vendor` directory is generated?
> 
> Done!

Yes if we want to AUTOMATE the dry-run usage, no if we only want to show the output to the user.

File: src/Commands/Update.php (hunk 1 above)
Comment (mjauvin) [reply src/Commands/Update.php]: > Yeah I think we'd have to, @mjauvin. That's the entire point of running a dry run stuck_out_tongue_winking_eye

@bennothommo Yes if we want to AUTOMATE the dry-run usage, no if we only want to show the output to the user.

**Pull request merged**